src/sentry/conf/server.py

```diff
@@ -112,6 +112,7 @@ def env(
 SENTRY_TRANSACTION_NAMES_REDIS_CLUSTER = "default"
 SENTRY_WEBHOOK_LOG_REDIS_CLUSTER = "default"
 SENTRY_ARTIFACT_BUNDLES_INDEXING_REDIS_CLUSTER = "default"
+SENTRY_INTEGRATION_ERROR_LOG_REDIS_CLUSTER = "default"
```

Comment:
Are you planning to use the `default` cluster in prod? Or did you wanna piggy back on an existing cluster. If you're going with option 2, I think you'll need to define `SENTRY_INTEGRATION_ERROR_LOG_REDIS_CLUSTER = "<whatever-cluster-youre-piggybacking>"` in prod settings like https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/prod.py#L930 and probably have the `getsentry` PR merged before this one.

Reply:
@barkbarkimashark do you have any clue what I'm supposed to put here https://github.com/getsentry/getsentry/pull/11243/files#r1275310032 like should it match whatever is defined in this sentry file?

Reply:
For posterity we talked about this and my changes [here](https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/prod.py#L505) and [here](https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/defaults.py#L1271) should be sufficient